### DISCUSSION TITLE
Need icon for showing 'history' in search

### DISCUSSION THREAD

**Discussion by Khushli** [twilio-labs/paste#1678] [Request: Icon] [answered]
Hi team,

We need an icon that can show historic searches by a users within the dropdown of search suggests. Something like what google has or similar. Open to other suggestions.
<img width="370" alt="Screen Shot 2021-07-19 at 12 13 27 PM" src="https://user-images.githubusercontent.com/87436655/126192845-25bc2baa-4c6d-4859-b157-982e7de4c59c.png">

![image](https://user-images.githubusercontent.com/87436655/126193027-3fe8a81e-5d8c-4bc2-96ef-2923f44dfc5d.png)

**Comment by andioneto** [comment]
Hi @Khushli, I just raised this with our design team and someone will be in touch within the next few days. Thank you for reaching out!

**Reply by Khushli** [reply]
Found an icon on streamline. Leaving the link here https://app.streamlinehq.com/icons/streamline-light/interface-essential/synchronize/synchronize-arrow-clock

**Comment by TheSisb** [comment]
Hi @Khushli,

We [added a ticket](https://issues.corp.twilio.com/browse/DSYS-2853) for this icon into our backlog.

**Comment by gloriliale** [comment, marked as answer]
Hi @Khushli, wanted to update and let you know we have released a HistoryIcon in Paste!
